hphp/CMakeLists.txt

```diff
@@ -28,6 +30,13 @@ if (EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/facebook")
                         ${CMAKE_MODULE_PATH})
 endif()
 
+find_package(PostgreSQL)
```

Comment:
Why? This doesn't belong here, it belongs in the extension config mechanism. (where it already is)